lib/cache/OperationWriter.js

```diff
@@ -0,0 +1,255 @@
+/*!
+ * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
+ */
+const database = require('bedrock-mongodb');
+const logger = require('../logger');
+const LRU = require('lru-cache');
+const {config, util: {BedrockError}} = require('bedrock');
+const {LruCache} = require('@digitalbazaar/lru-memoize');
+
+const PROMISE_QUEUE = new LruCache({max: 10, disposeOnSettle: true});
```

Comment:
What's the rationale for the max size here? Will the same LRU cache be servicing more than 10 concurrent requests at a time? Should the name of the promise queue be more targeted for which function it is for? Or is it intended to cover multiple different functions? It looks like it is only used in one place right now, so I recommend making its name more specific.